src/components/copy-to-clipboard.jsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable max-len */
+import React from "react";
+import ReactDOM from "react-dom";
+import Clipboard from "clipboard";
+
+export default class CopyToClipboard extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  componentDidMount() {
+    this.clipboard = new Clipboard(ReactDOM.findDOMNode(this.refs.copyToClipboard));
+  }
+
+  componentWillUnmount() {
+    this.clipboard.destroy();
+  }
+
+  render() {
+    return (
+      <button
+        ref="copyToClipboard"
```

Comment:
You can use the newfangled [ref callback](https://facebook.github.io/react/docs/more-about-refs.html#the-ref-callback-attribute) style and remove the `componentDidMount` implementation:

`ref={(el) => this.clipboard = el}`